neurodamus/gap_junction.py

```diff
@@ -66,6 +69,8 @@ def __init__(self, gj_conf, target_manager, cell_manager, src_cell_manager=None,
         super().__init__(gj_conf, target_manager, cell_manager, src_cell_manager, **kw)
         self._src_target_filter = target_manager.get_target(cell_manager.circuit_target,
                                                             src_cell_manager.population_name)
+        self.holding_ic_per_gid = None
+        self.SEClamp_current_per_gid = None
```

Comment:
totally nitpick, sorry, but would it be too bad to adhere to normal lowercase?
```suggestion
        self.seclamp_current_per_gid = None
```

Reply:
done